apis/extension/node.go

```diff
@@ -125,9 +140,45 @@ func GetNodeCPUSharePools(nodeTopoAnnotations map[string]string) ([]CPUSharedPoo
 	if err != nil {
 		return nil, err
 	}
+
+	cpusReserved := ""
+	if val, ok := nodeTopoAnnotations[CPUsReservedByNode]; ok {
+		cpusReserved = val
+	}
+
+	cpuSharePools = filterOutCpusByNodeAnno(cpuSharePools, cpusReserved)
+
 	return cpuSharePools, nil
 }
 
+// filterOutCpusByNodeAnno filter out cpus that reserved by annotation of node.
+func filterOutCpusByNodeAnno(cpuSharePools []CPUSharedPool, cpusReservedByNodeAnno string) []CPUSharedPool {
```

Comment:
removeNodeReservedCPUs might be more clear?

Reply:
en